spencer-buchness.html

```diff
@@ -0,0 +1,45 @@
+<html>
+<head>
+<title>CSS Timeline</title>
+<meta http-equiv="content-type" content="text/html; charset=UTF-8">
+<link rel="stylesheet" type="/spencer-buchness.css" href="spencer-buchness.css">
```

Comment:
Haha crap. I googled how to call in css and it had those two commands which
I thought were weird but it seemed to work for me. I'll fix it

On Tuesday, March 1, 2016, Tom Kraak <notifications@github.com> wrote:

> In spencer-buchness.html
> <https://github.com/betamore/fewd-css-exercise-1/pull/7#discussion_r54636241>
> :
>
> > @@ -0,0 +1,45 @@
> > +<html>
> > +<head>
> > +<title>CSS Timeline</title>
> > +<meta http-equiv="content-type" content="text/html; charset=UTF-8">
> > +<link rel="stylesheet" type="/spencer-buchness.css" href="spencer-buchness.css">
>
> there is no such "type" :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/betamore/fewd-css-exercise-1/pull/7/files#r54636241>.
>